src/locale/am.js

```diff
@@ -0,0 +1,23 @@
+//! moment.js locale configuration
+//! locale : Amharic [am]
+
+import moment from 'moment/dist/moment'
+
+export default moment.defineLocale('am', {
+    relativeTime: {
+      future: '%s ውስጥ',
+      past: '%s በፊት',
+      s: 'ጥቂት ሰከንዶ',
+      ss: '%d ሴኮንድ',
+      m: 'አንድ ደቂቃ',
+      mm: '%d ድቂቃዎች',
+      h: 'አንድ ሰዓት',
+      hh: '%d ሰዓታት',
+      d: 'አንድ ቀን',
+      dd: '%d ቀናት',
+      M: 'አንድ ወር',
+      MM: '%d ወራት',
+      y: 'አንድ ዓመት',
+      yy: '%d አመታት',
```

Comment:
I didn't see these in the translation sheet. I'm not certain, but the %d may be confusing here. Thankfully Amharic is left-to-right at least! Amharic has their own numeral system:
https://omniglot.com/language/numbers/amharic.htm

Reply:
I took these from the `time_x` rows in the translation spreadsheet. The only one that wasn't on there was the singular form of month but between google translate and other resources, I was able to put that together. Technically it should read for all singular forms like "a second" but roughly translating to "one second" also works.

For the numbers this is tricky because there are some resources I find that have 1-9 digits (no zero that I can find) but others like the official 2020 census docs in Amharic make it seem like using `%d` would be fine. I can add their own numeral system if needed.